src/main/java/wordsWithSpaces/WordsWithSpaces.java

```diff
@@ -0,0 +1,20 @@
+package wordsWithSpaces;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class WordsWithSpaces {
+    private static Pattern pattern = Pattern.compile("[A-Za-z]\\S*\\b");
```

Comment:
it's better to declare it as a constant: `private static final Pattern PATTERN`

Reply:
Added